tests/test_all_rules.py

```diff
@@ -696,3 +696,27 @@ def test_rule_backports_for_restricted_fields(self):
             err_msg = 'The following rules have min_stack_versions lower than allowed for restricted fields:\n'
             err_msg += invalid_str
             self.fail(err_msg)
+
+
+class TestRiskScoreMismatch(BaseRuleTest):
+    """Test that severity and risk_score fields contain corresponding values"""
+
+    def test_rule_risk_score_severity_mismatch(self):
+        invalid_list = []
+        risk_severity = {
+            "critical": 99,
```

Comment:
Should these by chance be a range?

Example:
`low: range(0,22)`

Then adjust the check to be `if risk_score not in risk_severity[severity]: ...`